### PR TITLE
Quieten Open311 errors

### PR DESCRIPTION
**Pull request by dracos**
This should stop the "read timeout" emails.
And also store the actual server error in the database for easier debugging.

### CODE REVIEW

File: perllib/FixMyStreet/SendReport/Open311.pm [hunk 1]
```diff
@@ -144,14 +144,11 @@ sub send {
             $self->success( 1 );
         } else {
             $result *= 1;
-            # temporary fix to resolve some issues with west berks
-            if ( $row->bodies_str =~ /2619/ ) {
-                $result *= 0;
-            }
+            $self->error( "Failed to send over Open311\n" ) unless $self->error;
+            $self->error( $self->error . "\n" . $open311->error );
```
Comment (dracos) [inline perllib/FixMyStreet/SendReport/Open311.pm]: Note: open311->error might not be set here, it's only set if non-200 response received (if bad 200 response received, it is `warn`ed but not set in the error attribute, I guess it probably should be).

Comment (davea) [inline perllib/FixMyStreet/SendReport/Open311.pm]: If it's not set will this cause the `uninitialised value` warning to be printed? Or is it simply an empty string if not set?

Comment (davea) [inline perllib/FixMyStreet/SendReport/Open311.pm]: Nevermind, I've just seen `default => ''` in `perllib/Open311.pm` 👍

**Comment by davea** [comment]
👍

File: perllib/FixMyStreet/SendReport/Open311.pm (hunk 1 above)
Comment (dracos) [inline perllib/FixMyStreet/SendReport/Open311.pm]: It should be the empty string, from `has error => ( is => 'rw', 'isa' => Str, default => '' );` so it should only add a newline at present.

**Pull request merged**